### PR TITLE
Added encoding option as a command-line argument

### LINKED ISSUE #21: Encoding error
I had to modify a few lines as was getting encoding exceptions

f = open(save_path + "." + extension, "w+", encoding="UTF-8")

and

with open(filepath, encoding="UTF-8") as fp:

to support UTF-8 encoding. Not sure if you want to add that and maybe an encoding option parameter in a newer version

### PR DESCRIPTION
**Pull request by yasircoskun**
Hi.

i'm a software developer from Turkey and we use genarelly utf-8 encoding for Turkish specific latin characters. I needed the project but i seen an error message in command-line when i try to use. After i see an [open issue](https://github.com/L4TTiCe/djangify/issues/21) (Closes #21) i wanted to contribute by generalizing the solution I used.

This is to be my first contribution, so if i maked a mistake please warn to me.

- Tested on Win10 (Python 3.10)
- imported `locale & encodings` (built-in modules)
- Comment lines added

- Sayonara

### CODE REVIEW

**Comment by yasircoskun** [comment]
- Tested on Win10 (Python 3.10)
- imported `locale & encodings` (built-in modules)
- Comment lines added

File: setup.cfg [hunk 1]
```diff
@@ -1,2 +1,2 @@
 [metadata]
-description-file = README.md
\ No newline at end of file
+description_file = README.md
```
Comment (amartya-dev) [inline setup.cfg:2]: Why is this change needed ?

File: djangify/djangify.py [hunk 2]
```diff
@@ -154,8 +162,16 @@ def main():
...
+    # use the locale encoding used for text data, according to user preferences.
+    # [Python Docs](https://docs.python.org/3/library/locale.html#locale.getpreferredencoding)
+    if (encoding is None or encodings.search_function(encoding) is None):
+        encoding = locale.getpreferredencoding()
+        
+
     logging.info("Directory : " + str(directory))
-    logging.info("app_name  : " + str(app_name))
+    logging.info("app_name  : " + str(APP_NAME))
+    logging.info("Encoding  : " + str(encoding))
```
Comment (amartya-dev) [inline djangify/djangify.py]: Let us not log encoding, we do not need the user to know about this, although `log.debug` could be used here.

File: setup.cfg (hunk 1 above)
Comment (yasircoskun) [reply setup.cfg:2]: warn

Comment (amartya-dev) [reply setup.cfg:2]: Thanks

File: djangify/djangify.py (hunk 2 above)
Comment (yasircoskun) [reply djangify/djangify.py]: i'm changing this but I don't understand what's different for "Encoding". Why user is informed about "app_name" and "directory" again, even though he set this value himself. Why that isnt applies for "Encoing"  too.

**Review by amartya-dev — APPROVED** [review]
